src/Application/Application.csproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Ardalis.GuardClauses" />
-    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" />
+    <PackageVersion Include="AutoMapper" />

```

Comment:
This should be `<PackageReference Include="AutoMapper" />`. PackageVersion is used in **Directory.Packages.props**.

Reply:
Oh... my bad... I overlooked this. Thanks